app/Importer/AssetImporter.php

```diff
@@ -21,11 +22,24 @@ protected function handle($row)
         parent::handle($row);
 
         if ($this->customFields) {
+            $input = array();
```

Comment:
@marcusmoore @spencerrlongg  Sorry to invoke you, just want to leave a reminder here to see on monday if needed. We set the CustomFields here, but they aren't tied to an asset, so I had to manually retrieve the model to access the validation rules and validate the fields before we create the asset to save it. So I create a custom validator that supposedly shows the error and then stop the import. But I can only 'skip' silently the rows that doesn't meet the criteria, without an actual error.